MapboxNavigation/NavigationViewController.swift

```diff
@@ -12,6 +12,13 @@ import Mapbox
  */
 public typealias ContainerViewController = UIViewController & NavigationComponent
 
+/**
+ The position of speed limit view and floating buttons, including the overview, mute and feedback report in a navigation view.
+ */
+public enum MapOrnamentPosition {
+    case topLeading, topTrailing
```

Comment:
Remember to add a documentation comment for both of these cases. Otherwise, the jazzy docset will show an enumeration with no cases.

Reply:
Thanks for the suggestion, I'll add the two cases to documentation.